sources/packages/backend/apps/workers/src/controllers/assessment/_tests_/e2e/assessment.controller-verifyAssessmentCalculationOrder.e2e-spec.ts

```diff
@@ -269,4 +277,362 @@ describe("AssessmentController(e2e)-verifyAssessmentCalculationOrder", () => {
       isReadyForCalculation: true,
     });
   });
+
+  it("Should sum the grants and awards from two past applications with different offering intensities and SFAS/SAIL applications data when the applications are for the same student and program year.", async () => {
```

Comment:
the description of `two past applications` is tightly coupled with data prepared for test. It can be generic like `past applications`